lib/in_app_payments.dart

```diff
@@ -40,6 +40,9 @@ typedef BuyerVerificationSuccessCallback = void Function(
     BuyerVerificationDetails result);
 typedef BuyerVerificationErrorCallback = void Function(ErrorInfo errorInfo);
 
+typedef CardOnFileBuyerVerificationSuccessCallback = void Function(
```

Comment:
@jguze I think we need a better name since this can be used not only for CardOnFile. 

How about `BuyerVerificationWithPaymentSourceIdSuccessCallback`?

Reply:
@hukid Ah I made a similar comment. What do you think of `BuyerVerificationFlowSuccessCallback`

Possibly too similar, in which case I prefer your choice

Reply:
If we choose a simple name, I'd rename the existing `BuyerVerificationSuccessCallback` to `BuyerVerificationWithCardEntrySuccessCallback`

Reply:
@shivamsaviant Let's go with what I proposed here: https://github.com/square/in-app-payments-flutter-plugin/pull/145/files#r600937108

Reply:
Okay @jguze